docs/viewlifecycle.md

```diff
@@ -303,19 +310,23 @@ The `CollectionView` creation lifecycle can go down two paths depending on
 whether the collection is populated or empty. The below table shows the order of
 rendering events firing:
 
-| Order |           Event          |
-| :---: |--------------------------|
-|   1   |      `before:render`     |
-|  2*   |  `before:render:empty`   |
-|  2+   | `before:render:children` |
-|   3   |    `before:add:child`    |
-|   4   |       `add:child`        |
-|  5*   |      `render:empty`      |
-|  5+   |     `render:children`    |
-|   6   |          `render`        |
-|   7   |      `before:attach`     |
-|   8   |         `attach`         |
-|   9   |       `dom:refresh`      |
+| Order |           Event          |                Arguments                |
+| :---: |--------------------------|-----------------------------------------|
+|   1   |      `before:render`     | `this` - collection view being rendered |
+|  2*   |  `before:render:empty`   | `this` - collection view being rendered |
+|       |                          |    `view` - empty view being rendered   |
+|  2+   | `before:render:children` | `this` - collection view being rendered |
+|   3   |    `before:add:child`    | `this` - collection view being rendered |
```

Comment:
I just checked in one app and this and view points to the same object (collection) in onBeforeAddChild
http://codepen.io/blikblum/pen/XjLRbB